src/content/events/2024/hackathon-march-2024/usa-uw.md

```diff
@@ -8,18 +8,76 @@ endDate: '2024-03-20'
 endTime: '13:00-06:00'
 locationName: University of Washington
 address: 1607 NE Pacific St, Seattle, WA
-locationURL: https://www.washington.edu/maps/#!/hseb
-locationLatLng: [47.651960, -122.310820]
+locationURL: https://www.washington.edu/maps/#!/hsrr
+locationLatLng: [47.649292, -122.308287]
 ---
 
 Local event to be held at the University of Washington.
 
-Contact:
+## Contact:
 
 - [<i class="fab fa-slack"></i> Carson Miller](https://nfcore.slack.com/team/U04T0N2631P)
 
-Exact Room HSEB 424
+## Agenda
 
-## Acknowledgments
+<div class="table-responsive">
+    <table class="table table-hover table-sm table-bordered">
+        <thead>
+            <tr>
+                <th>Time</th>
+                <th>Mon. 18 Mar., 2024</th>
+                <th>Tue. 19 Mar., 2024</th>
+                <th>Wed. 20 Mar., 2024</th>
+            </tr>
+            </thead>
+            <tbody>
+            <tr>
+                <td>9:00</td>
+                <td background-color:navy; rowspan="1">Welcome and Setup</td>
+                <td background-color:navy; rowspan="1">Daily Start</td>
+                <td background-color:navy; rowspan="1">Daily Start</td>
+            </tr>
+                <td>10:00</td>
+                <td rowspan="6">Hack!</td>
+                <td rowspan="6">Hack!</td>
+                <td rowspan="6">Hack!</td>
+            </tr>
+            <tr>
+                <td>11:00</td>
+            </tr>
+            <tr>
+                <td>12:00</td>
+            </tr>
+            <tr>
+                <td>13:00</td>
+            </tr>
+            <tr>
+                <td>14:00</td>
+            </tr>
+            <tr>
+                <td>15:00</td>
+            </tr>
+            <tr>
+                <td>16:00</td>
+                <td background-color:navy; rowspan="1">Daily sync</td>
+                <td background-color:navy; rowspan="1">Daily sync</td>
+                <td background-color:navy; rowspan="1">Final wrap-up</td>
+            </tr>
+        </tbody>
+    </table>
+</div>
 
-This local event is being kindly supported by the University of Washington
+## Organisational Information
+
+### Registration
+
+Please use the main registration form to sign up and select the location you want. If the Seattle location does not appear as an option (this node was recently added), please let [Carson Miller](#contact) know that you will be attending, so that we can adjust our plan based on the number of people.
```

Comment:
```suggestion
Please use the main registration form to sign up and select the location you want. If the Seattle location does not appear as an option (this node was recently added), please let [Carson Miller](#contact) know that you will be attending, so that we can adjust our plan based on the number of people.
```

`#contact` above is re-directing back to the webpage

Reply:
Updated to point to my Slack account now!